src/main/java/com/bartek/Medical/service/HospitalService.java

```diff
@@ -0,0 +1,46 @@
+package com.bartek.Medical.service;
+
+import com.bartek.Medical.exception.InvalidNameHospitalException;
+import com.bartek.Medical.hospitalRepository.HospitalRepository;
+import com.bartek.Medical.mapper.DoctorMapper;
+import com.bartek.Medical.mapper.HospitalMapper;
+import com.bartek.Medical.model.DoctorDTO;
+import com.bartek.Medical.model.Hospital;
+import com.bartek.Medical.model.HospitalDto;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
```

Comment:
pola robimy finalne jesli mozemy i uzywamy requiredArgs do wszystkich serwisow, controllerow sie to odnosi